### PR TITLE
Publish.css Integration into EbullientWorks

### PR DESCRIPTION
**Pull request by sigrunixia**
This update brings Publish.css Integration into Ebullientworks. 
It include the update released yesterday to checkboxes.

It also removes .theme-dark .print as this is no longer needed in the next release of Obsidian, so this may be better tagged as a next version release. 

Thread is #25 .

### CODE REVIEW

File: src/fragments/01a-colors-mixins.scss
Comment (ebullient) [inline src/fragments/01a-colors-mixins.scss:1]: Holy macaroni!! =)

I have one ask (which may be odd).. can you keep the base color groupings together at the top? Rainbow + grayscale. When I fuss with colors (which I don't do terribly often), I like the spectrum together.

I would be especially happy if they were in the same order as the original: 
red, yellow, green, cyan, blue, purple, pink .. then the gray, with all of the other mixins following.

File: images/publish-showcase/sample-calloits-light.png
Comment (ebullient) [inline images/publish-showcase/sample-calloits-light.png:1]: maybe a filename typo. ;)

File: images/publish-showcase/sample-calloits-dark.png
Comment (ebullient) [inline images/publish-showcase/sample-calloits-dark.png:1]: maybe a filename typo. ;)

File: src/fragments/03-core-theme-light.scss [hunk 1]
```diff
@@ -1,6 +1,19 @@
-.theme-dark .print,
```
Comment (ebullient) [inline src/fragments/03-core-theme-light.scss:1]: Before removing this, can you verify what happens when you print when you're in dark mode? Last I checked, the results were a little off.. So I basically flipped dark-mode to light mode if print was involved..

File: src/fragments/04-headings-hr-tags.scss [hunk 2]
```diff
@@ -74,14 +84,14 @@ h1,
 .markdown-rendered,
 .workspace,
 .print {
-  .is-live-prevew .HyperMD-header .cm-header::before,
+  .is-live-preview .HyperMD-header .cm-header::before,
```
Comment (ebullient) [inline src/fragments/04-headings-hr-tags.scss:87]: 🤦‍♀️

File: src/fragments/04-headings-hr-tags.scss [hunk 3]
```diff
@@ -74,14 +84,14 @@ h1,
...
   h1::before,
   h2::before,
   h3::before,
   h4::before,
   h5::before,
   h6::before {
-    content: "\00a7\00a0";
+    content: '\00a7\00a0';
```
Comment (ebullient) [inline src/fragments/04-headings-hr-tags.scss:94]: 🤦‍♀️

File: src/fragments/04-headings-hr-tags.scss [hunk 4]
```diff
@@ -142,7 +157,8 @@ body:not(.ebullientworks-nix-tags) {
     --tag-color: var(--color-accent-2);
     --tag-color-hover: var(--tag-3);
   }
-  &.theme-dark .print,
+
+  &.theme-dark,
```
Comment (ebullient) [inline src/fragments/04-headings-hr-tags.scss:161]: There was a distinction here, as the settings applied only to the dark theme when printing: `&.theme-dark .print`

File: src/fragments/12-calendar-day-planner-tracker.scss [hunk 5]
```diff
@@ -12,7 +12,7 @@
     color: var(--gray-light-5);
   }
 }
-.theme-dark .print,
+.theme-dark,
```
Comment (ebullient) [inline src/fragments/12-calendar-day-planner-tracker.scss:15]: Again.. this originally applied only to dark-mode when printing...

File: src/fragments/13-timelines.scss [hunk 6]
```diff
@@ -16,7 +16,7 @@
     border: 4px solid var(--primary-accent-4);
   }
 }
-.theme-dark .print,
+.theme-dark,
```
Comment (ebullient) [inline src/fragments/13-timelines.scss:19]: only dark mode when printing..

**Review by ebullient — CHANGES_REQUESTED** [review]
More in a bit..

File: images/publish-showcase/sample-calloits-dark.png
Comment (sigrunixia) [reply images/publish-showcase/sample-calloits-dark.png:1]: Corrected

File: images/publish-showcase/sample-calloits-light.png
Comment (sigrunixia) [reply images/publish-showcase/sample-calloits-light.png:1]: Corrected :)

**Review by sigrunixia — COMMENTED** [review]
Should be good to go. But I could always miss something. :)

File: src/fragments/15-table.scss [hunk 7]
```diff
@@ -1,35 +1,34 @@
...
+  --ebw-table-header-alt: var(--primary-accent-4);
 }
-.theme-dark .print,
-.theme-light {
-  --table-header-color: var(--gray-dark-7);
+
+@mixin table-light-dark-print {
   --table-header-background: var(--primary-accent-1);
-  --table-header-border-color: var(--primary-accent-3);
-  --table-header-weight: var(--font-semibold);
```
Comment (ebullient) [inline src/fragments/15-table.scss:21]:
```
  --table-header-border-color: var(--primary-accent-3);
  --table-header-weight: var(--font-semibold);
```

These were dropped?

File: src/publish.css.map
Comment (ebullient) [inline src/publish.css.map:1]: Did you mean to include this?

File: theme.css
Comment (ebullient) [inline theme.css:1]: Don't include this in the commit. Will come from the release..

File: src/fragments/15-table.scss (hunk 7 above)
Comment (sigrunixia) [reply src/fragments/15-table.scss:21]: What in the world. Yup. Fixing now.

File: src/publish.css.map
Comment (sigrunixia) [reply src/publish.css.map:1]: Never know if I should include those. Adding those to the "do not send, junk files" list.

File: theme.css
Comment (sigrunixia) [reply theme.css:1]: Deleted :)

**Pull request closed without merging**